### ISSUE TITLE
Get into fixing up problems found by QuantifiedCode 

### ISSUE THREAD

**Issue by psi29a** [twisted/txmongo#197]
https://www.quantifiedcode.com/app/project/9f65cb1b729e4f7eaad1fa4048169009

I plan on working my way through this, normally this should not impact end-users.

**Comment by IlyaSkriblovsky** [comment]
seems like they are mostly related to the coding style

**Comment by psi29a** [comment]
Exactly, somethings we can't touch because of either Twisted or PyMongo. ;)
